ckan/templates/home/snippets/search.html

```diff
@@ -5,8 +5,7 @@
   <form class="module-content search-form" method="get" action="{% url_for controller='package', action='search' %}">
     <h3 class="heading">{{ _("Search data") }}</h3>
     <div class="search-input form-group search-giant">
-      <label for="field-main-search">{% block header_site_search_label %}{{ _('Search Datasets') }}{% endblock %}</label>
-      <input id="field-main-search" type="text" class="form-control" name="q" value="" autocomplete="off" placeholder="{% block search_placeholder %}{{ placeholder }}{% endblock %}" />
+      <input aria-label="{% block header_site_search_label %}{{ _('Search datasets') }}{% endblock %}" id="field-main-search" type="text" class="form-control" name="q" value="" autocomplete="off" placeholder="{% block search_placeholder %}{{ placeholder }}{% endblock %}" />
```

Comment:
@gocemitevski I think "Search datasets" needs to be dynamic, meaning if a user is on the Datasets page it should be "Search Datasets", on the Organizations page it should be "Search organizations", etc.

Reply:
@klikstermkd this is not related to the contextual search form for datasets/organization/groups... in `templates/snippets/search_form.html`, but the home page widget, so it is no problem to have it hard-coded. Datasets are the only thing you can search for, from within the widget.